src/components/editor/ValueConstraints.jsx

```diff
@@ -42,6 +44,23 @@ class ValueConstraints extends Component {
   }
 }
 
+ValueConstraints.propTypes = {
+    defaults: PropTypes.oneOfType([PropTypes.array, PropTypes.string]),
```

Comment:
did you get these from Christina's json schema?  (e.g. https://github.com/LD4P/sinopia_sample_profiles/blob/profile-spec/schemas/property-template.json ) ?

Reply:
In part, I also looked at the existing tests and component code to determine the props.